blocks/components/editable/style.scss

```diff
@@ -10,4 +10,15 @@
 	&:focus {
 		outline: none;
 	}
+
+	&.is-empty {
+		position: relative;
+	}
+
+	&.is-empty:before {
```

Comment:
After looking it over, I believe it only exists on `<input>`, and `<textarea>`.  I forgot that we are using content editable lol. `::placeholder` and various vendor prefixes are often used to create consistency across the various browsers for the way they display the placeholder text, so I think `::before` definitely makes the most sense now, disregard my comments :).  I don't think you can set the `content` as a property for `::placeholder`, so even if using it made sense it won't help us at all.